log4j-core/src/test/resources/log4j2-3182.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration status="WARN" packages="org.apache.logging.log4j.core.pattern" monitorInterval="3">
+    <properties>
+        <property name="PATTERN">%d{yyyy-MM-dd HH:mm:ss.SSS} |-%-5level [%thread] %c [%L] -| %msg%n</property>
+    </properties>
+
+    <appenders>
+        <Console name="CONSOLE" target="system_out">
+            <PatternLayout pattern="${PATTERN}" />
+        </Console>
+
+        <Kafka name="KafkaLog" topic="acm">
+            <PatternLayout pattern="%msg"/>
+            <Property name="bootstrap.servers">10.20.10.11:9092</Property>
```

Comment:
I don't think this is an OS/network-independent solution. How do you know that the Kafka server is on `10.20.10.11`? Further, doesn't this test depend on a running Kafka server? If so, where do we start it?

Reply:
It doesn't depend on a running server. This is for mocking a thread leak during monitorinterval file on change suituation.  Even the kafka ip address is not available, it won't affect this testcase